rhizome/host/lib/vm_setup.rb

```diff
@@ -268,6 +268,9 @@ def setup_taps_6(gua, nics)
     nics.each do |nic|
       r "ip -n #{q_vm} addr replace #{guest_intrusion.shellescape} dev #{nic.tap}"
 
+      dns_ipv4 = NetAddr::IPv4Net.parse(nic.net4.split("/").first + "/26").nth(2).to_s
```

Comment:
Seems a little weird to hard code as /26 here in duplication. I'm not sure why you need that to get the second address from the base? Could you even make it a /32?

Reply:
The address added is actually a /32. Since the subnets we allocate are /26 and I want to uniformly allocate the  second address, I'm finding the base first, and then the second address. Proper way would be to pass it with the prep.json. This was easier to do. I'll refactor this later since now this blocks @velioglu 's PR.

Reply:
Actually, I refactored it already.